docs/extensions/guides/ipc.md

```diff
@@ -112,20 +112,17 @@ As this example shows: the channel names *must* be the same.
 It should also be noted that "listeners" and "handlers" are specific to either `LensRendererExtension` and `LensMainExtension`.
 There is no behind the scenes transfer of these functions.
 
-If you want to register a "handler" you would call `Ipc.IpcMain.handleRpc(...)` instead.
+If you want to register a "handler" you would call `IpcMain.getInstance().handle(...)` instead.
 The cleanup of these handlers is handled by Lens itself.
 
-`Ipc.IpcRenderer.broadcastIpc(...)` and `Ipc.IpcMain.broadcastIpc(...)` sends an event to all renderer frames and to main.
+Calling either `IpcRenderer.getInstance().broadcast(...)` or `IpcMain.getInstance().broadcast(...)` sends an event to all renderer frames and to main.
```

Comment:
Since we used backticks on `main` everywhere else, the one reference here at the end of the sentence should have them too.